src/senaite/core/upgrade/v02_05_000.py

```diff
@@ -293,3 +295,43 @@ def _remove_action(type_info, action_id):
     index = actions.index(action_id)
     type_info.deleteActions([index])
     return _remove_action(type_info, action_id)
+
+
+def import_workflow(tool):
```

Comment:
Please add the `@upgradestep(product, version)` decorator to `import_workflow` function. Workflows are modified by other add-ons quite often.